packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+@import "@itwin/itwinui-css/scss/variables";
+
+.select-project {
+  .title-section {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 16px;
+    justify-content: space-between;
+    flex-wrap: wrap-reverse;
+
+    .inline-input-with-button {
+      display: flex;
+      flex-direction: row;
+
+      .iui-input-container.iui-inline > .iui-input:last-child {
+        min-width: 300px;
+        padding-right: 61px;
+      }
+
+      .iui-button {
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+        margin-left: -50px;
+      }
+    }
+
+    .iui-input-container.iui-inline {
+      margin: 0;
+    }
+  }
+
+  > .iui-tabs-wrapper {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    //position: absolute;
+    width: 100%;
```

Comment:
could remove the width, theres a horizontal scrollbar added because of the margin 
![image](https://user-images.githubusercontent.com/11051042/133854507-c39f7e4e-f30c-4e64-b38a-6ecc33247214.png)